### PR TITLE
[MXNET-703] Show perf info for TensorRT during tests

### PR DESCRIPTION
**Pull request by KellenSunderland**
## Description ##
This PR makes sure perf information printed during TensorRT test runs is correctly displayed when run in CI.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

**Comment by KellenSunderland** [comment]
@vandanavk Resolved.

**Comment by vandanavk** [comment]
@marcoabreu @lebeg @aaronmarkham for review

File: ci/docker/runtime_functions.sh [hunk 1]
```diff
@@ -741,7 +741,7 @@ unittest_ubuntu_tensorrt_gpu() {
     export MXNET_STORAGE_FALLBACK_LOG_VERBOSE=0
     export LD_LIBRARY_PATH=/work/mxnet/lib:$LD_LIBRARY_PATH
     python tests/python/tensorrt/lenet5_train.py
-    nosetests-3.4 $NOSE_COVERAGE_ARGUMENTS --with-xunit --xunit-file nosetests_trt_gpu.xml --verbose tests/python/tensorrt/
+    nosetests-3.4 $NOSE_COVERAGE_ARGUMENTS --with-xunit --xunit-file nosetests_trt_gpu.xml --verbose --nocapture tests/python/tensorrt/
```
Comment (lebeg) [inline ci/docker/runtime_functions.sh]: Isn't `--nocapture` for disabling output to std?

**Comment by KellenSunderland** [comment]
No by default it's disabled and this enables it.  Terrible API.

**Comment by KellenSunderland** [comment]
@nswamy Could you have a look at this PR?

**Comment by Roshrini** [comment]
@marcoabreu @sandeep-krishnamurthy This PR seems ready to merge. Can you take a look once and merge if it looks good?

**Pull request merged**